game/scripts/vscripts/items/cave_rune_hasted_doubledamage.lua

```diff
@@ -0,0 +1,26 @@
+LinkLuaModifier("modifier_rune_haste", LUA_MODIDIER_MOTION_NONE)
+LinkLuaModifier("modifier_rune_doubledamage", LUA_MODIDIER_MOTION_NONE)
+
+item_cave_rune_hasted_doubledamage = class(ItemBaseClass)
+
+function item_cave_rune_hasted_doubledamage:GetIntrinsicModifierName()
+
+  return "modifier_rune_haste", "modifier_rune_doubledamage"
+
+end
+
+function item_cave_rune_hasted_doubledamage:OnSpellStart()
+	local caster = self:Getcaster()
+
+	caster:AddNewModifier(caster, self, "modifier_rune_haste", {duration = 30.0})
+
+	caster:AddNewModifier(caster, self, "modifier_rune_doubledamage", {duration = 30.0})

```

Comment:
Durations need to be read from KV. See `GetSpecialValueFor` method.
The double damage will likely need a custom modifier, so that the damage increase amount can be exposed to KV.